### PR TITLE
✨ Create Login Button for Spotify

### LINKED ISSUE #17: Spotify Connect Button
# User Story
As a user,
I want to login with Spotify
so that I don't need to register at another service and make use of my running Spotify subscription

# Acceptance Criteria
- Onclick Property

# Screenshot
image

### PR DESCRIPTION
**Pull request by joshua-meyerrr**
This Resolves #17.

Create Button as Link. Takes:
- button text as children
- url as prop
- optional className for styling

[Component in Storybook](https://scrive-app-login-button-gmtuvv.herokuapp.com/storybook/?path=/story/component-button--spotify-login)

### CODE REVIEW

File: src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export default function SpotifyLoginButton({
+  children,
+  className,
+  url,
+}: SpotifyLoginButtonProps): JSX.Element {
+  return (
+    <Link to={url} className={`${styles.button} ${className}`}>
+      <SpotifyLogo width={32} height={32} />
+      <p className={styles.button__text}>{children}</p>
+      {/* replace with Button Typo */}
```
Comment (marwinburesch) [inline src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx:21]: whut?

File: src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  url: string;
+};
+
+export default function SpotifyLoginButton({
+  children,
+  className,
+  url,
+}: SpotifyLoginButtonProps): JSX.Element {
+  return (
+    <Link to={url} className={`${styles.button} ${className}`}>
```
Comment (marwinburesch) [inline src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx:18]: button? I thought it's an anchor :D :D 
Maybe name the class `loginLink`?

**Review by marwinburesch — COMMENTED** [review]
good stuff, yes 🍌

File: src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx (hunk 1 above)
Comment (joshua-meyerrr) [reply src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx:21]: Isn't it approriate to leave comments for my forgetful future self? Eg to replace something with a component when it has been created?

File: src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx (hunk 2 above)
Comment (joshua-meyerrr) [reply src/app/components/SpotifyLoginButton/SpotifyLoginButton.tsx:18]: Had to laugh out loud haha - thanks. I'll take it, Sir 🤠

**Pull request merged**